### PR TITLE
Use tus chunksize from backend

### PR DESCRIPTION
**Pull request by JammingBen**
## Description
Instead of using the tus chunksize from the web config, we now use the chunksize coming from the backend (if set).

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Technical debt
- [ ] Tests

### CODE REVIEW

File: packages/web-runtime/src/services/uppyService.ts [hunk 1]
```diff
@@ -43,11 +43,7 @@ export class UppyService {
     tusExtension: string
     headers: { [key: string]: string }
   }) {
-    const chunkSize =
-      tusMaxChunkSize > 0 && uploadChunkSize !== Infinity
-        ? Math.max(tusMaxChunkSize, uploadChunkSize)
-        : uploadChunkSize
-
+    const chunkSize = tusMaxChunkSize > 0 ? tusMaxChunkSize : uploadChunkSize
```
Comment (kulmann) [inline packages/web-runtime/src/services/uppyService.ts:46]:
```suggestion
    const chunkSize = tusMaxChunkSize || uploadChunkSize
```
should do the same

**Review by kulmann — APPROVED** [review]

File: changelog/unreleased/enhancement-tus-backend-chunksize [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Enhancement: Use tus chunksize from backend
+
+Instead of using the tus chunksize from the web config, we now use the chunksize coming from the backend (if set).
```
Comment (pascalwengerter) [inline changelog/unreleased/enhancement-tus-backend-chunksize:3]: Can we remove the config chunksize entirely then?

Comment (kulmann) [reply changelog/unreleased/enhancement-tus-backend-chunksize:3]: if we can have a sane fallback value without config, then yes

Comment (JammingBen) [reply changelog/unreleased/enhancement-tus-backend-chunksize:3]: The current fallback is `infinity`, I'm not sure if this can be seen as "sane" 😅

**Review by kulmann — APPROVED** [review]

**Pull request merged**